tests/foreman/longrun/test_inc_updates.py

```diff
@@ -36,109 +35,104 @@
+    return entities.LifecycleEnvironment(name='DEV', organization=module_sca_manifest_org).create()
 
 
 @pytest.fixture(scope='module')
-def qe_lce(module_entitlement_manifest_org, dev_lce):
+def qe_lce(module_sca_manifest_org, dev_lce):
     qe_lce = entities.LifecycleEnvironment(
-        name='QE', prior=dev_lce, organization=module_entitlement_manifest_org
+        name='QE', prior=dev_lce, organization=module_sca_manifest_org
     ).create()
     return qe_lce
 
 
 @pytest.fixture(scope='module')
-def rhel7_sat6tools_repo(module_entitlement_manifest_org, module_target_sat):
+def sat_client_repo(module_sca_manifest_org, module_target_sat):
     """Enable Sat tools repository"""
-    rhel7_sat6tools_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+    sat_client_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
         basearch=DEFAULT_ARCHITECTURE,
-        org_id=module_entitlement_manifest_org.id,
+        org_id=module_sca_manifest_org.id,
         product=PRDS['rhel'],
-        repo=REPOS['rhst7']['name'],
-        reposet=REPOSET['rhst7'],
+        repo=REPOS['rhsclient7']['name'],
+        reposet=REPOSET['rhsclient7'],
         releasever=None,
     )
-    rhel7_sat6tools_repo = entities.Repository(id=rhel7_sat6tools_repo_id).read()
-    assert rhel7_sat6tools_repo.sync()['result'] == 'success'
-    return rhel7_sat6tools_repo
+    sat_client_repo = entities.Repository(id=sat_client_repo_id).read()
+    assert sat_client_repo.sync()['result'] == 'success'
+    return sat_client_repo
 
 
 @pytest.fixture(scope='module')
-def custom_repo(module_entitlement_manifest_org):
+def custom_repo(module_sca_manifest_org):
     """Enable custom errata repository"""
     custom_repo = entities.Repository(
         url=settings.repos.yum_9.url,
-        product=entities.Product(organization=module_entitlement_manifest_org).create(),
+        product=entities.Product(organization=module_sca_manifest_org).create(),
     ).create()
     assert custom_repo.sync()['result'] == 'success'
     return custom_repo
 
 
 @pytest.fixture(scope='module')
-def module_cv(module_entitlement_manifest_org, rhel7_sat6tools_repo, custom_repo):
+def module_cv(module_sca_manifest_org, sat_client_repo, custom_repo):
     """Publish both repos into module CV"""
     module_cv = entities.ContentView(
-        organization=module_entitlement_manifest_org,
-        repository=[rhel7_sat6tools_repo.id, custom_repo.id],
+        organization=module_sca_manifest_org,
+        repository=[sat_client_repo.id, custom_repo.id],
     ).create()
     module_cv.publish()
     module_cv = module_cv.read()
     return module_cv
 
 
 @pytest.fixture(scope='module')
-def module_ak(module_entitlement_manifest_org, module_cv, custom_repo, module_lce_library):
+def module_ak(
+    module_sca_manifest_org, module_cv, custom_repo, module_lce_library, module_target_sat
+):
     """Create a module AK in Library LCE"""
     ak = entities.ActivationKey(
```

Comment:
This now imports `module_target_sat` fixture already
```suggestion
    ak = module_target_sat.api.ActivationKey(
```

Reply:
this will consider separately